requirements_common.txt

```diff
@@ -21,3 +21,4 @@ tensorboard>=1.15.0
 tqdm>=4.30
 scipy
 torchio>=0.18.68
+wandb
```

Comment:
You may wish to fix a specific version on this.

Reply:
Just out of curiosity, why specify a version? Is it because updates to wandb may deprecate some of the functions and we'd have to keep updating those in ivadomed as well?